src/main/java/core/basesyntax/Randomable.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public interface Randomable {
+    ColorSupplier colorSupplier = new ColorSupplier();
+    Random random = new Random();
+    void generateRandomParameters();
+}
```

Comment:
Any sense of having such interface. 

1. interface shouldnt' have state 
2. exposing public random and colorsupplier are bad practice
3. abstract method doesn't have any sense, cause it's responsibility of FigureSupplier

```suggestion
```